src/tokenized-line.coffee

```diff
@@ -8,8 +8,26 @@ LeadingWhitespaceRegex = /^\s*/
 TrailingWhitespaceRegex = /\s*$/
 RepeatedSpaceRegex = /[ ]/g
 CommentScopeRegex = /(\b|\.)comment/
+TabCharCode = 9
+SpaceCharCode = 32
+SpaceString = ' '
+TabStringsByLength = {
+  1: ' '
+  2: '  '
+  3: '   '
+  4: '    '
+}
+
```

Comment:
Arrays should be generally faster for numeric loads, trade-off is mostly just style.
ps.
I've posted bench on jsperf http://jsperf.com/array-vs-obj-num-load, though it may be too naive, but it shows no difference in chrome, so looks like V8 is smart enough to use array for object in this case.